Expose an option to disable cookies

kamilszostak opened the pull request:
Fixes #335

SergeyKanzhelev commented on API-reference.md:
```diff
@@ -281,6 +281,10 @@ Values that control how the telemetry data is sent.
+
+        // If true, the SDK will not store or read any data from cookies.
+        // Default: false
+        disableCookies: boolean;
```
can you move it closer to `cookieDomain` setting so they will be sitting together.

kamilszostak replied:
Done

pelikhan commented:
Can there be a flag to disable local storage as well?

kamilszostak commented:
I updated the pull request. 

* Exposed and option to disable session and local storage
* CanUseLocal and CanUseSession storage methods now cache the results, so it will be slightly faster
* Renamed configuration fields : `isCookieUseDisabled` and `isStorageUseDisabled`

pelikhan commented:
Looks great!
